components/calendar/src/date.rs

```diff
@@ -129,21 +130,36 @@ impl<A: AsCalendar> Date<A> {
     ) -> Result<Self, DateError> {
         let inner = calendar
             .as_calendar()
-            .date_from_codes(era, year, month_code, day)?;
+            .from_codes(era, year, month_code, day)?;
         Ok(Date { inner, calendar })
     }
 
+    /// Construct a date from a fixed day and some calendar representation
```

Comment:
"fixed" is Reingold jargon, please say more in the docs

Reply:
Well, `rd` is also jargon, maybe spell it out as `to_rata_die`? Or `to_day_number`?

Reply:
rata_die is fine, that's unambigous field-standard jargon.

Reply:
so change the method names or not?

Reply:
`to_rata_die` for public API. Rata Die is a well defined term. There are many different ways to calculate day numbers, and the Rata Die is a specific choice.